app/src/main/res/values/text_styles.xml

```diff
@@ -9,6 +9,10 @@
         <item name="fontPath">@string/AvenirNext_DemiBold</item>
     </style>
 
+    <style name="AvenirNext_Bold" parent="android:TextAppearance">
+        <item name="fontPath">@string/AvenirNext_Bold</item>
```

Comment:
That's `Demi_Bold`, you need to add the `Bold`. I will send the file by slack, I have already converted all the font files from the mac format to `.ttf`